finetune/base.py

```diff
@@ -435,6 +441,29 @@ def _define_placeholders(self):
         self.Y = tf.stop_gradient(tf.placeholder(tf.float32, [None, self.target_dim or 1])) # classification targets
         self.do_dropout = tf.placeholder(tf.float32)  # 1 for do dropout and 0 to not do dropout
 
+    def generate_text(self, max_length=None, seed_text=''):
```

Comment:
These arguments should probably be fixed.  It's more likely the seed_text is passed in as a positional argument than it is that max_length is.